src/components/Rating.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const List = styled.li`
+  backgroundColor: '#e8f5e9',
+  margin: '1rem',
+  padding: '1rem',
+`;
+
+const Heading = styled.h3`
+  borderBottom: '1px solid rgba(0, 0, 0, 0.1)',
+  fontWeight: 300,
+  margin: 0,
+
+`;
+export const Rating = ({ rating }) => {
+  return (
+    <List key={rating._id}>
+      <Heading>
+        {rating.author} reported this number as "{rating.callType}"
+      </Heading>
+      <p>{rating.message}</p>
+      <time
+        dateTime={new Date(rating.createdAt).toISOString}
```

Comment:
You need to call `.toISOString()`, not just pass it as a reference ;)

Reply:
btw `<time>` is built in component of react?

Reply:
`<time>` was a new element in HTML5 :) https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time